habitat_baselines/rl/ddppo/algo/ddp_utils.py

```diff
@@ -29,6 +31,46 @@
 )
 
 
+def is_slurm_job() -> bool:
+    return SLURM_JOBID is not None
+
+
+def is_slurm_batch_job() -> bool:
+    return is_slurm_job() and os.environ.get("SLURM_JOB_NAME", "bash") not in (
```

Comment:
Maybe, add a small comment or doc string how does it work. Not obvious connection `bash` and batch job property.